Articles/Setup/Server-Setup/permissions.md

```diff
@@ -0,0 +1,85 @@
+---
+v8-equivalent: "https://github.com/umbraco/UmbracoDocs/blob/main/Getting-Started/Setup/Server-Setup/permissions.md"
+versionFrom: 9.0.0
+verified-against: alpha-4
+state: partial
+updated-links: true
+---
+
+# File and folder permissions
+<!--- THIS NEEDS TO BE REVISITED
+:::note
+To ensure a stable and smoothly running Umbraco installation, these permissions need to be set correctly. These permissions should be setup before or during the installation of Umbraco.
+
+The main account that requires 'modify' file permissions to be set on the folders below, is the account used by the Application Pool Identity for the IIS website. Usually IIS APPPOOL\appPoolName or a specific local account or in some circumstances Network Service - If in doubt, ask your server admin / hosting company. Additionally the IUSR account and IIS_IUSRS account only require 'read only' access to the site's folders.
+
+Generally when developing locally with Visual Studio or WebMatrix, permissions do not need to be strictly applied.
+:::
+
+:::note
+If you have any specific static files / media items / etc then you should add the appropriate permissions accordingly.
+The permissions documentation as it is should allow you to run a plain Umbraco install successfully, the rest.. is up to you! 👍
+::: 
+-->
```

Comment:
If this entire section needs to be revisited - does it then make sense to wait with adding this article to the docs? 🤔 
Or what do you think?

Reply:
If we can save the table that I updated in this file, it would save us some work, when we are revisiting the article